tensorflow_graphics/projects/mesh_rcnn/structures/mesh.py

```diff
@@ -0,0 +1,172 @@
+# Copyright 2020 The TensorFlow Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Data Structure for Meshes"""
+
+import tensorflow as tf
+
+from tensorflow_graphics.geometry.convolution import utils
+from tensorflow_graphics.projects.mesh_rcnn.util import padding
+
+
+class Meshes:
+  """Wrapper for batching of Meshes with unequal size."""
+
+  def __init__(self, vertices, faces, batch_sizes=None):
+    """
+    Contstructs the wrapper object from nested lists of multiple vertices and
+    faces.
+
+    Internally, vertices and faces are padded and packed into a single 2D
+    tensor. The object provides accessors for different representations,
+    including:
+
+    * List: List of all unpadded vertices and faces tensors. This is intended as
+        input format only. However, it is possible to retrieve a flat list of
+        all vertices and faces tensor via `get_unpadded()`.
+    * Padded: Tensor of shape `[A1, ..., An, max(N), 3]`, where max(N) is the
+        size of the largest vertex or face list in all batch dimensions. All
```

Comment:
size of the largest vertex (or face) list.

This helps clarify max is over num vertices for vertex tensor, not max(num_verts, num_faces)